src/main/java/teammates/common/datatransfer/FeedbackSessionAttributes.java

```diff
@@ -114,6 +114,22 @@ public FeedbackSessionAttributes(String feedbackSessionName, String courseId, St
         this.respondingInstructorList = instructorList;
         this.respondingStudentList = studentList;
     }
+    
+    public String getCourseId() {
+        return courseId;
+    }
+    
+    public String getFeedbackSessionName() {
+        return feedbackSessionName;
+    }
+    
+    public String getStartTime() {
+        return TimeHelper.formatTime(startTime);
+    }
+    
+    public String getEndTime() {
+        return TimeHelper.formatTime(endTime);
+    }
```

Comment:
I would appreciate input from others here.
Would it be better to give these more detailed method names?
As this class is used in many places, it may not be good to assume that people won't call `getStartTime()` when wanting a Date object (especially if we ever set the variable to `private`).